plugin/trino-http-event-listener/src/test/java/io/trino/plugin/httpquery/TestHttpQueryListener.java

```diff
@@ -274,20 +270,18 @@ public void testContentTypeDefaultHeaderShouldAlwaysBeSet()
     public void testHttpHeadersShouldBePresent()
             throws Exception
     {
-        EventListener querylogListener = factory.create(new HashMap<>(){{
-                put("http-event-listener.connect-ingest-uri", server.url("/").toString());
-                put("http-event-listener.log-completed", "true");
-                put("http-event-listener.connect-http-headers", "Authorization: Trust Me!, Cache-Control: no-cache");
-            }});
+        EventListener querylogListener = factory.create(Map.of(
+                "http-event-listener.connect-ingest-uri", server.url("/").toString(),
+                "http-event-listener.log-completed", "true",
+                "http-event-listener.connect-http-headers", "Authorization: Trust Me!, Cache-Control: no-cache"));
 
         server.enqueue(new MockResponse().setResponseCode(200));
 
         querylogListener.queryCompleted(queryCompleteEvent);
 
-        checkRequest(server.takeRequest(5, TimeUnit.SECONDS), new HashMap<>() {{
-                put("Authorization", "Trust Me!");
-                put("Cache-Control", "no-cache");
-            }}, queryCompleteEvent);
```

Comment:
i lost `queryCompleteEvent`.
i don't understand how CI figured this out, as this test method is not `@Test` annotated (https://github.com/trinodb/trino/pull/10635), yet it failed (https://github.com/trinodb/trino/runs/4826429554?check_suite_focus=true)